lib/editorInfoAtCursor.ts

```diff
@@ -48,7 +48,13 @@ export const editorInfoAtCursor =
         const yIndex = line - section.startLine - startLine
         if (yIndex < 0) return noop
 
-        const clueIndexes = clueInfosForPosition(data.clues, { col: index, index: yIndex })
+        let clueIndexes = {} as any
+        try {
+          clueIndexes = clueInfosForPosition(data.tiles, data.clues, { col: index, index: yIndex })
+        } catch (e) {
+          // tile not found
+        }
```

Comment:
swapped to a different version of clueInfosForPosition which throws an error rather than returning undefined if nothing found.